mito-ai/src/Extensions/AiChat/ChatMessage/CodeBlock.tsx

```diff
@@ -43,6 +43,11 @@ const CodeBlock: React.FC<ICodeBlockProps> = ({
         navigator.clipboard.writeText(codeWithoutMarkdown)
     }
 
+    const getCodeCellID = () => {
```

Comment:
Should this be using the active cell? I think this is supposed to use the cell id that is stored in the chat history manager.